home/tests.py

```diff
@@ -94,9 +94,9 @@ class TestAnswersManipulations:
         def answers(self):
             profile = Profile.objects.first()
             question = Question.objects.get(id=3)
-            ans1 = Answer(profile=profile, question=question, content='Answer 1', publish_date=datetime(2021, 4, 1,),
```

Comment:
Please remove all the unnecessary whitespace changes from the PR - it makes reviewing it harder.